dali/base/dasds.cpp

```diff
@@ -7847,6 +7804,29 @@ StringBuffer &CCovenSDSManager::getLocks(StringBuffer &out)
     return out.length() ? out : out.append("No current locks");
 }
 
+void CCovenSDSManager::getLocks(CMessageBuffer &mb)
+{
+    CLockDataHelper helper;
+    unsigned lockCount=0;
+    mb.clear();
+    mb.append(lockCount);
+
+    CHECKEDCRITICALBLOCK(lockCrit, fakeCritTimeout);
+    SuperHashIteratorOf<CLockInfo> iter(lockTable.queryBaseTable());
+    iter.first();
+    while (iter.isValid())
+    {
+        CLockInfo &lockInfo = iter.query();
+        if (lockInfo.lockCount())
+            lockInfo.serializeLockData(helper, mb);
+        lockCount++;
```

Comment:
Should I not count when 0 == lockInfo.lockCount() ? If not count, why the lockInfo is in the iter?